Update collection creation documentation to reflect the latest deployment changes (#117)
OpenTermsArchive/docs: content/collections/create.en.md

Note: user must have the right to `sudo`.

### On GitHub

#### Create a fine-grained repo-scoped token to allow OTA-Bot to create issues on the declarations repository

- Log in on GitHub as OTA-Bot user
- Go to <https://github.com/settings/personal-access-tokens/new>
- Set the “Token name”: “Issue creation on `<collection_name>` collection”
- Set expiration at one year later

> - Select OpenTermsArchive organization in “Resource owner”

Reviewer: ```suggestion
- Select “OpenTermsArchive“ organization in “Resource owner”
```

- Select “Only select repositories” in “Repository access”
- Select “`<collection_name>`-declarations” in “Select repositories”
- Select “Issues — Access: Read and write” in “Permissions” → “Repository permissions”
- Validate with “Generate token and request access”
- Copy the token and back it up in the shared passwords repository

#### Validate the token

- Log in on GitHub with an admin account of the OpenTermsArchive organization
- Go to <https://github.com/organizations/OpenTermsArchive/settings/personal-access-token-requests>
- Select the pending request
- Approve it

#### Add an SSH key created for this collection

- Create a new SSH key: `ssh-keygen -t ed25519 -C bot@opentermsarchive.org`
- Copy the public and private keys and back them up in the shared passwords repository
- Log in on GitHub as OTA-Bot user
- Go to <https://github.com/settings/ssh/new>
- Set the “Title”: “`<collection_name>` collection”
- Paste the public key in “Key”
- Validate with “Add SSH Key”

> ### On Brevo OpenTermsArchive account

Reviewer: ```suggestion
### On Open Terms Archive Brevo account
```

- Create an SMTP key to allow sending error notifications by email:

>   - Log in on Brevo as OpenTermsArchive admin user

Reviewer: ```suggestion
  - Log in on Brevo as Open Terms Archive admin user
```

  - Go to <https://app.brevo.com/settings/keys/smtp>
  - “Generate a new SMTP key”
  - Set the SMTP key name “Name your SMTP key”: `<collection_name>` collection”
  - Validate with “Generate”
  - Copy the key and back it up in the shared passwords repository

### On your machine

> #### Encrypt sensistive data

Reviewer 2: :wink:
```suggestion
#### Encrypt sensitive data
```

> - Create a new vault key to encrypt sensitive data

Reviewer: How do I do that?
